lua/complementree/sources.lua

```diff
@@ -166,6 +165,109 @@ local function lsp_completedone(completed_item)
   end
 end
 
+--
+
+local os = string.lower(jit.os)
+local is_linux = (os == 'linux' or os == 'osx' or os == 'bsd')
+local os_sep = is_linux and '/' or '\\'
+local os_path = '[' .. os_sep ..'%w+%-%.%_]*$'
+
+M.filepath_matches = function(opts)
```

Comment:
Could you switch to the other style:
```suggestion
function M.filepath_matches(opts)
```